ee/tabby-webserver/src/cron/db/gitlab.rs

```diff
@@ -37,18 +50,25 @@ async fn refresh_repositories_for_provider(
             service
                 .update_provider_status(provider.id.clone(), false)
                 .await?;
-            warn!(
-                "GitLab credentials for provider {} are expired or invalid",
-                provider.display_name
-            );
+            context
```

Comment:
@boxbeam could you help make a macro that write to `warn!` and `stderr_writeline` at same time?